docs/IntentReader.md

```diff
@@ -55,8 +55,10 @@ The PubSub plugin has to be integrated with FCA to call IntentReader with PubSub
 
 ### Usage
 
-With the configuration and setup in place, we can send messages to FCA through any pubsub mechanism like Websocket, etc. First, we have to initialise a pubsub connection, then publish and subscribe topics to send or receive messages from FCA.
-We can perform any action or task with intentReaders and with the help of multiple handlers. For performing a particular task, give the required task name in the "task" field and the corresponding task parameters required to perform the task.
+* The sender (which can be a framework, server, or application) needs to have the PubSub plugin integrated into it.
```

Comment:
The sender could also just be a user opening a web socket connection to the PubSub client FCA is subscribed to. Not sure if we want to be that specific about the sender or just generically say the external sender configured to send PubSub messages to the FCA PubSub client - or something along those lines.

Reply:
updated as   "The external sender needs to have the PubSub plugin integrated into it."